cmd/cluster/openstack/create.go

```diff
@@ -34,17 +34,13 @@ func BindOptions(opts *RawCreateOptions, flags *pflag.FlagSet) {
 }
 
 func bindCoreOptions(opts *RawCreateOptions, flags *pflag.FlagSet) {
-	// TODO(stephenfin): This is unnecessary given the information should already be in clouds.yaml. We should deprecate and remove it.
-	flags.StringVar(&opts.OpenStackCredentialsFile, "openstack-credentials-file", opts.OpenStackCredentialsFile, "Path to the OpenStack credentials file (optional)")
```

Comment:
I think I put the TODO on the wrong line. We might want to keep this if we want to allow users to specify an exact `clouds.yaml` and not rely on auto-discovery? Thoughts?

Reply:
I'm thinking of the use case where a customer provides a clouds.yaml without the CAcert because they have the CAcert of the cloud part of their trusted CA (e.g. on a company laptop, where CA trust is handled by the IT admins. They don't need the CAcert, but the HCP would need it).
In fact I'm thinking this whole PR is dangerous and we can still offer that flexibility for now. I also want to discuss with others in the team to know what they think.

cc @mandre @pierreprinetti @mdbooth @MaysaMacedo